content/en/news/releases/1.4.x/announcing-1.4.3/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Announcing Istio 1.4.3
+linktitle: 1.4.3
+subtitle: Patch Release
+description: Istio 1.4.3 patch release.
+publishdate: 2020-01-08
+release: 1.4.3
+aliases:
+    - /news/announcing-1.4.3
+---
+
+This release includes bug fixes to improve robustness and user experience. This release note describes what’s different between Istio 1.4.2 and Istio 1.4.3.
+
+{{< relnote >}}
+
+## Bug fixes
+
+- **Fixed** an issue where Mixer creates too many watches, overloading `kube-apiserver` ([Issue 19481](https://github.com/istio/istio/issues/19481)).
+- **Fixed** an issue with injection when pod has multiple containers without exposed ports ([Issue 18594](https://github.com/istio/istio/issues/18594)).
+- **Fixed** overly restrictive validation of `regex` field ([Issue 19212](https://github.com/istio/istio/pull/19212)).
```

Comment:
Do we want to change `Issue` to `Pull Request` in the text? This line and next? Not sure if it actually more accurate or not.